components/FundingPlatform/QuestionBuilder/ReviewerManagementTab.tsx

```diff
@@ -69,39 +95,47 @@ export const ReviewerManagementTab: React.FC<ReviewerManagementTabProps> = ({
       });
     },
     onSuccess: async () => {
-      await queryClient.invalidateQueries({ queryKey: ["program-reviewers", programId, chainID] });
-      toast.success("Reviewer added successfully");
+      await queryClient.invalidateQueries({
+        queryKey: ["program-reviewers", programId, chainID],
```

Comment:
use builder keys or have a queryKeys files nd put all the keys there so that "program-reviewers" with the params are defined once in the code